python/modules/IcePy/Operation.cpp

```diff
@@ -44,13 +44,24 @@ namespace IcePy
     class Operation
     {
     public:
-        Operation(const char*, PyObject*, int, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*);
+        Operation(
+            const char*,
+            const char*,
+            PyObject*,
+            int,
+            PyObject*,
+            PyObject*,
+            PyObject*,
+            PyObject*,
+            PyObject*,
+            PyObject*);
```

Comment:
Added an extra parameter to `IcePy::Operation`

The old code relied upon there being a way to convert between the mapped-operation and `current.operation`.
`python:identifier` makes it impossible to go from one to other, so now we have to pass both the slice and mapped names across the barrier between C++ and Python:
```diff
-    MyInterface._op_op = IcePy.Operation('OP', Ice.OperationMode.Normal, False, None, (), (), (), None, ())
+    MyInterface._op_op = IcePy.Operation('OP', 'MAPPED', Ice.OperationMode.Normal, False, None, (), (), (), None, ())
```